apps/dashboard/src/components/workflow-editor/steps/in-app/in-app-action.tsx

```diff
@@ -154,7 +155,7 @@ const ConfigureActionPopover = (props: ComponentProps<typeof PopoverTrigger> & {
   return (
     <Popover modal={true}>
       <PopoverTrigger {...rest} />
-      <PopoverContent className="max-w-72">
+      <PopoverContent className="max-w-72" side="bottom" align="end">
```

Comment:
the actions popover form is aligned with the end of the action button. Otherwise, when text is typed, it will jump